docs/user/tpm-authentication.md

```
@@ -0,0 +1,232 @@
+For Kubernetes deployments, add TPM CA certificates via ConfigMap and configure the API server:
+
+1. **Create ConfigMap with TPM CA certificates:**
+
+```bash
+kubectl create configmap tpm-ca-certs \
+  --from-file=infineon-root-ca.pem \
+  --from-file=infineon-intermediate-ca.pem \
+  --namespace=flightctl
+```
+
+2. **Update the API server configuration to include TPM CA paths:**
```

markdownlint-cli2 (0.17.2)

Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/1/1

(MD029, ol-prefix)